pkg/proxy/service_test.go

```diff
@@ -173,8 +174,24 @@ func TestServiceToServiceMap(t *testing.T) {
 				}
 			}),
 			expected: map[ServicePortName]*BaseServiceInfo{
-				makeServicePortName("ns1", "load-balancer", "port3", v1.ProtocolUDP): makeTestServiceInfo("172.16.55.11", 8675, "UDP", 0),
-				makeServicePortName("ns1", "load-balancer", "port4", v1.ProtocolUDP): makeTestServiceInfo("172.16.55.11", 8676, "UDP", 0),
+				makeServicePortName("ns1", "load-balancer", "port3", v1.ProtocolUDP): makeTestServiceInfo("172.16.55.11", 8675, "UDP", 0, func(info *BaseServiceInfo) {
+					info.loadBalancerStatus = v1.LoadBalancerStatus{
```

Comment:
This adds a lot of vertical space to an otherwise terse test, making it hard to read.  It seems this could be:

```
info.loadBalancerStatus.Ingress = []v1.LoadBalancerIngress{{IP: "10.1.2.4"}}
```

This applies to all cases and should make this already ridiculously long list a bit shorter